README.rst

```diff
@@ -286,6 +286,67 @@ input file. When this setting is missing, the default value ``true`` is used.
 .. |--xunit report.xml| replace:: ``--xunit report.xml``
 .. _`--xunit report.xml`: https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#xunit-compatible-result-file
 
+Parse for Polyspace Failures
+----------------------------
+
+To use the Polyspace checker, a TSV file must be exported from Polyspace.
+You can find instructions on exporting TSV files in the Polyspace documentation.
+
+Exporting the TSV File
+~~~~~~~~~~~~~~~~~~~~~~
+
+The following commands export a TSV file from Polyspace:
+
+.. code-block:: bash
+
+    polyspace-results-export -format csv -results-dir <resultsFolder> <export_options>
+    # or
+    polyspace-results-export -format csv -host <hostName> -run-id <runID> <export_options> <polyspace_access_options>
+
+The csv format outputs tab-separated values (TSV).
+Each result in the TSV format consists of tab-separated information in columns such as ID,
+Family, Group, Color, Check, and so on.
```

Comment:
Would it be helpful to document all column titles here? And mention that, starting from R2024a, line and column information are also exported.

Reply:
I can say that starting from R2024a columns: ID, Family, Group, Color, New, Check, Information, Function, File, Status, Severity, Comment, Key, Line and Col are available

Reply:
Resolved in dd3b13e7d55e4336828d41761d9d4c67902b505c

Reply:
You're not documenting the distinction between pre and post R2024a. This could be useful information.

Reply:
Is this sufficient? aee25d04f9ea9afcc3fa124a444fd3adb900d136